app/lib/forms/choose_your_npq.rb

```diff
@@ -35,6 +35,12 @@ def options
     end
 
     def after_save
+      wizard.store["targeted_delivery_funding_eligibility"] = funding_eligibility_calculator.targeted_funding[:targeted_delivery_funding]
+      wizard.store["tsf_primary_plus_eligibility"] = funding_eligibility_calculator.targeted_funding[:tsf_primary_plus_eligibility]
+      wizard.store["tsf_primary_eligibility"] = funding_eligibility_calculator.targeted_funding[:tsf_primary_eligibility]
+      wizard.store["funding_eligiblity_status_code"] = funding_eligibility_calculator.funding_eligiblity_status_code
```

Comment:
This is so much better to have it decided once and then persisted onwards. Does this get used everywhere instead of calculating on the fly now?

Reply:
So there is one place where it doesnt seem to work but trying to investigate why although not sure will fix in this PR. It caused a lot of pain actually trying to workout why.